scripts/compose-dev.yml

```diff
@@ -21,6 +21,8 @@ services:
 
   postgres:
     image: centos/postgresql-95-centos7 
+    ports:
+    - '5432:5432'
```

Comment:
Map `5432` to the localhost port, so that local postgres tools can be used, and the database restore process can be run locally.